package.json

```diff
@@ -126,6 +126,7 @@
     "test-flow": "flow .",
     "test-cov": "nyc --require=flow-remove-types/register --reporter=text-summary --reporter=lcov --cache run-s test-unit test-render test-query",
     "prepublish": "in-publish && run-s build-dev build-min || not-in-publish",
+    "preinstall": "ps -fp $PPID | grep -q yarn || (echo '\\033[0;33mNPM not supported, please use Yarn.\\033[0m'; exit 1) && yarn cache clean",
```

Comment:
I suspect this command will fail on Windows, since it doesn't have `ps`, `grep`, etc.